scripts/install_deps_req.sh

```diff
@@ -1,21 +1,34 @@
 #!/bin/bash
-sudo apt-get -qq update
-# install Eigen 3, Boost and TBB
-sudo apt-get --yes --force-yes install cmake libeigen3-dev libboost-serialization-dev libboost-filesystem-dev libboost-test-dev libboost-program-options-dev libboost-thread-dev libboost-regex-dev
-# install google tests for libcmaes
-sudo apt-get --yes --force-yes install libgtest-dev autoconf automake libtool libgoogle-glog-dev libgflags-dev
+OS=$(uname)
+echo "Detected OS: $OS"
+
+if [ $OS = "Darwin" ]; then
+    echo "Installing for OSX via brew"
+    brew install cmake eigen3 boost
+    brew install autoconf automake
+else
+    echo "Installing for Ubuntun via apt-get"
+    sudo apt-get -qq update
+    # install Eigen 3, Boost and TBB
+    sudo apt-get --yes --force-yes install cmake libeigen3-dev libboost-serialization-dev libboost-filesystem-dev libboost-test-dev libboost-program-options-dev libboost-thread-dev libboost-regex-dev
+    # install google tests for libcmaes
+    sudo apt-get --yes --force-yes install libgtest-dev autoconf automake libtool libgoogle-glog-dev libgflags-dev
+fi
 
 # save current directory
 cwd=$(pwd)
 # create install dir
 mkdir -p install
 
-# do libgtest fix for libcmaes
-cd /usr/src/gtest
-sudo mkdir -p build && cd build
-sudo cmake ..
-sudo make
-sudo cp *.a /usr/lib
+# do libgtest fix for libcmaes (Linux only)
+if [ $OS = "Darwin" ]; then
```

Comment:
If the comment is correct, this should be `if ! [ $OS = "Darwin" ]; then` right?